src/recallcard/randomcard.php

```diff
@@ -22,17 +22,29 @@
 }
 if ($type == "lesson") {
 	$lesson = get_lesson($getid);
-	$group = $conn->query("SELECT * FROM `group` WHERE `id` = '{$data_lesson['group']}'");
-	$data_group = $group->fetch_array();
+	$stmt = $conn->prepare("SELECT * FROM `group` WHERE `id` = ?");
+	$stmt->bind_param("i", $data_lesson['group']);
+	$stmt->execute();
+	$group = $stmt->get_result();

```

Comment:
It looks like `$data_group` variable disappears. Could you bring it back?

Reply:
Fixed.